user/caribou/template.py

```diff
@@ -0,0 +1,48 @@
+pkgname = "caribou"
+pkgver = "0.4.21"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-gtk2-module"]
```

Comment:
```suggestion
configure_args = ["--disable-gtk2-module", "--disable-static"]
```
and drop the -static subpkg